python/composio/tools/toolset.py

```diff
@@ -135,12 +135,14 @@ class ComposioToolSet(WithLogger):  # pylint: disable=too-many-public-methods
 
     _runtime: str = "composio"
     _description_char_limit: int = 1024
+    _action_name_char_limit: t.Optional[int] = None
```

Comment:
Consider adding a docstring for the `_action_name_char_limit` class variable to explain its purpose and default behavior. For example:
```python
# Maximum length for action names. None means no limit is applied.
_action_name_char_limit: t.Optional[int] = None
```